### PR TITLE
automate irp by default on new conductor services

### PR DESCRIPTION
**Pull request by hspearman**
although we want to give the option to opt-out, let's default to true for irp automation.

### CODE REVIEW

**Review by teresabocchini — APPROVED** [review]

File: workflow-templates/conductor-on-main.yaml [hunk 1]
```diff
@@ -25,3 +25,4 @@ jobs:
     with:
       env: production
       notify: true
+      automate_irp: true
```
Comment (jonhoman) [inline workflow-templates/conductor-on-main.yaml:28]: @hspearman What does this option do? Automatically freeze deploys?

Comment (hspearman) [reply workflow-templates/conductor-on-main.yaml:28]: yes, it's what triggers these message: 🧊  »» app «« has been frozen - triggered by Conductor 🤖

Comment (hspearman) [reply workflow-templates/conductor-on-main.yaml:28]: opt-in by default, with opt-out optional.

Comment (jonhoman) [reply workflow-templates/conductor-on-main.yaml:28]: Got it. Thanks.

**Pull request merged**